docs/proposals/algorithms/multi-edge-inference/proposal.md

```diff
@@ -0,0 +1,104 @@
+# Multi-task Joint Inference for Ianvs
+
+## Motivation
+Nowadays,  the inference of lifelong learning is somehow a single task approach, which will first vote for the task suited for the sample, and then the model trained for this task will be used for inference. This approach works well when the model for the task is easy to train. However, for the perception of mobile agents, more complex models are usually used. Besides,  as shown in figure 1, an empirical study concluded that no single model maintains the best accuracy even for one second. So we think the joint inference will be better.
+
+<img src="Fig1_Heatmap of normalized mAP on a clip of 1 second.png" width="400" />
+
+​														Fig 1. Heatmap of normalized mAP on a clip of 1 second
+
+
+We have noticed that autonomous driving is one of the important applications of edge AI. We consider the image object detection task on moving vehicles as the typical online model inference scenario. Recently, how to cooperate with resources on the edge and cloud to provide support for autonomous driving applications has become an important topic.  
+
+### Goals
+
+The goals of the multi-task joint inference for Ianvs are:
+
+- Adding an example named multi-task joint inference in Ianvs 
+  - choose tasks to realize joint inference according to various scenarios dynamically
+    - the characteristics of vehicle movement, the scenarios faced by autonomous vehicles are complicated
+  - fuse the output of multi tasks
+- Verifying the effectiveness of the joint inference in Object Detection
+
+## Proposal
+### Scope
+
+- Providing a new example for Ianvs
+
+## Design Details
+### Implementation
+
+Task, A triplet: Task Allocation +  Task samples + Task model 
+
+The architecture is shown in the below figure. The red box shows the work we are going to do. 
```

Comment:
The overall architecture is clear. But where is inavs components and origin ianvs components? The archtecture can be cooperated with lifelong learning lib for ianvs.

Reply:
We only found the single-task example and the incremental learning example. This is why we did not consider the lifelong learning lib for ianvs. In fact, this project does not contain the training phase. We would like first to show how to **infer** with multiple tasks (the inference phase). The enhancement of the tasks, i.e., lifelong **learning**, will be seen as future work.

Reply:
We see in the architecture, it is still marked sedna. Recently, lifelong learning is now available for usage in a dev branch and related introduction is conducted on the routine meeting in the past two week. Please consider using the lifelong learning lib in ianvs accordingly~  Ianvs components can be integrated in the architecture now.